.sparkswap/config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Sparkswap configuration for use inside the container. (Note the rpcCertPath is relative to being inside the container)
```

Comment:
Worth noting that this config is for using the broker-cli inside the container specifically.